source/android/adaptivecards/src/main/java/io/adaptivecards/objectmodel/CardElementType.java

```diff
@@ -23,6 +23,7 @@ public enum CardElementType {
   Image,
   ImageSet,
   Media,
+  MetaData,
```

Comment:
Same as shared model comment

Reply:
@jwoo-msft it is defined as a type here: https://adaptivecards.io/explorer/Metadata.html

Reply:
Removed